devices/src/virtio/mmio.rs

```diff
@@ -183,6 +183,22 @@ impl MmioDevice {
         }
     }
 
+    fn reset(&mut self) {
+        assert!(!self.device_activated);
```

Comment:
Don't do `assert` in release code. I suggest logging an error and moving on.

Reply:
done

Reply:
Please remove the leftover `assert!` too.

Reply:
sorry for my mistake, fixed:)